test/util.js

```diff
@@ -47,3 +47,37 @@ describe("parseSearchParams()", function() {
         assert.deepStrictEqual(result, { hello: "hello world", hello2: "hello world2" });
     });
 });
+
+describe("nDigitsText()", function() {
+    it("should be able to add filler characters to small numbers", () => {
+        let result = ripeCommons.nDigitsText(9);
+        assert.deepStrictEqual(result, "09");
+
+        result = ripeCommons.nDigitsText(5, 3);
+        assert.deepStrictEqual(result, "005");
+    });
+
+    it("should do nothing to big enough numbers", () => {
+        let result = ripeCommons.nDigitsText(50);
+        assert.deepStrictEqual(result, "50");
+
+        result = ripeCommons.nDigitsText(5000, 4);
+        assert.deepStrictEqual(result, "5000");
+    });
+
+    it("should cut huge numbers to the desired size", () => {

```

Comment:
I'm not sure this behavior makes much sense. Shouldn't it throw an error instead?